frontend/sw360-portlet/src/main/webapp/js/main.js

```diff
@@ -240,9 +240,11 @@ function deleteConfirmed(confirmMessage, confirmCallback) {
         content: confirmMessage,
         confirmButtonClass: 'btn-info',
         cancelButtonClass: 'btn-danger',
-        confirm: function() {
-            confirmCallback();
-        }
+        buttons: {
```

Comment:
You are totally right:

old version:
![withcancel](https://cloud.githubusercontent.com/assets/1187050/25481838/5507c7a0-2b4f-11e7-8215-461ab1a1d588.png)

new version:
![withoutcancel](https://cloud.githubusercontent.com/assets/1187050/25481843/5b410af0-2b4f-11e7-8f58-560d08c3a771.png)

I will fix this.